src/components/Button/Button.tsx

```diff
@@ -70,7 +70,7 @@ export class Button extends React.Component<IButtonProps, IButtonState> implemen
         {
           'aria-label': ariaLabel,
           'aria-labelledby': ariaLabel ? null : labelId,
-          'aria-describedby': ariaDescription ? ariaDescriptionId : description ? descriptionId : null,
+          'aria-describedby': ariaDescription ? ariaDescriptionId : description ? descriptionId : nativeProps['aria-describedby'] ? nativeProps['aria-describedby'] : null,
```

Comment:
it's not quite that simple (from what I can see, as I didn't write the original). It checks for `ariaDescription` and if that is not null then it uses  `ariaDescriptionId` for the attribute. Same thing with description.  Let me try to push the code I wrote yesterday. Github is giving me credential issues.